src/sentry_envelope.c

```diff
@@ -17,6 +17,9 @@ typedef enum {
 } endpoint_type_t;
 
 typedef enum {
+    // When loading an envelope from disk, we don’t really want to parse it,
+    // but just pipe it through as a raw buffer
+    ENVELOPE_ITEM_TYPE_ENVELOPE,
```

Comment:
Let's please find a way to not represent envelopes as items. It's totally fair to have a flag or raw payload, as you do this below with `is_raw_envelope`, and direct it to the store endpoint.